modules/bg96/cellular_bg96.c

```diff
@@ -136,6 +138,95 @@ static bool appendRatList( char * pRatList,
 
 /*-----------------------------------------------------------*/
 
+static bool _Cellular_CreateUdpSocketConnectMutex( cellularModuleSocketContext_t * pBg96SocketContext )
+{
+    bool status = false;
+
+    status = PlatformMutex_Create( &pBg96SocketContext->udpSocketConnectMutex, false );
+
+    return status;
+}
+
+/*-----------------------------------------------------------*/
+
+static void _Cellular_DestroyUdpSocketConnectMutex( cellularModuleSocketContext_t * pBg96SocketContext )
+{
+    PlatformMutex_Destroy( &pBg96SocketContext->udpSocketConnectMutex );
+}
+
+/*-----------------------------------------------------------*/
```

Comment:
Can you elaborate the purpose of these two functions?
They are static functions and contain only one line.
Unless, there is something to be considered. I would like to suggest to remove these functions.

Reply:
Good suggestion. I removed them.